backend/src/dbutil.ts

```diff
@@ -1,18 +1,126 @@
-import * as dt from "../schema/heat_pump_water_heater";
+import * as dt from "../schema/appliance";
 
-const dummyWaterHeater: dt.HeatPumpWaterHeater = {
-  brandName: "Rheem",
-  modelNumber: "PROPH40 T0 RH120-M",
+export function findWaterHeater(size: number, uef: number, fhr: number) {
+  return dummyWaterHeaters;
+}
+
+export function findDryer(noise: number, cef: number, capacity: number) {
+  return dummyDryers;
+}
+
+const dummyWaterHeaters: dt.HeatPumpWaterHeater[] = [{
+  brandName: "Brand A",
+  modelNumber: "GEH50DFEJ2RA",
+  modelType: "701460",
```

Comment:
The modelType for all of these fake appliances should be one of the allowed enums.

Reply:
Looks like the field `modelType` is misnamed and should be renamed to `modelVariant` instead as per the spec.